src/com/javacore/lesson3/HomeWork.java

```diff
@@ -0,0 +1,31 @@
+package com.javacore.lesson3;
+
+
+public class HomeWork {
+
+    public static void main(String[] args) {
+
+        SuperArrayList<String> superArrayList = new SuperArrayList<>();
```

Comment:
As you have interface for your implementation better will be:
`BestList <String> superArrayList = new SuperArrayList<>();`